Sources/MarathonCore/Package.swift

```diff
@@ -14,7 +14,7 @@ public struct Package {
 }
 
 extension Package: Equatable {
-    public static func ==(lhs: Package, rhs: Package) -> Bool {
+    public static func == (lhs: Package, rhs: Package) -> Bool {
```

Comment:
It causes the warning like `warning: Operator Function Whitespace Violation: Operators should be surrounded by a single whitespace when defining them. (operator_whitespace)`.
We can disable it. What do you think? For me, it makes sense based on the [Advanced Operators](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/AdvancedOperators.html) section.